multiqc/multiqc.py

```diff
@@ -689,8 +690,10 @@ def run(
                 except AttributeError:
                     pass
 
-        except UserWarning:
```

Comment:
We should still capture this for now, though maybe show a depreciation log message.

Reply:
Done!